analyses/golang/matcher_test.go

```diff
@@ -0,0 +1,82 @@
+package golang_test
+
+import (
+	"os"
+	"path/filepath"
+	"testing"
+
+	"github.com/fabric8-analytics/cli-tools/analyses/driver"
+	"github.com/fabric8-analytics/cli-tools/analyses/golang"
+)
+
+type isSupportedManifestTestcase struct {
+	Name    string
+	Want    bool
+	Matcher driver.StackAnalysisInterface
+}
+
+type depsTreeTestCase struct {
+	ManifestFile string
+	Want         string
+	Matcher      driver.StackAnalysisInterface
+}
+
+func (tc isSupportedManifestTestcase) TestMatcher_IsSupportedManifest(t *testing.T) {
```

Comment:
It is not so common in golang to use _. Can you just stick to the below format?
```suggestion
func (tc isSupportedManifestTestcase) isSupportedManifest(t *testing.T) {
```

Reply:
Ack.